### ISSUE TITLE
Multiple select incompatible with styled items

### ISSUE THREAD

**Issue by cherrydev** [valor-software/ng2-select#241]
If you place HTML inside of the `text` attribute of the SelectItem it will display the raw HTML code on the selected item when multiple selections are enabled.  I recommend a separate `template` attribute on SelectItem that supports an angular template.  The `template` would be used to render the list items when available and the existing `text` would be displayed (escaped) as a fallback and for selected items in multiple selection mode.

**Comment by rahul71** [comment]
We should be able to pass an optional parameter called value along with text. Value would be set as the selected item in case its passed in. Should be an easy change to make. Is the correct way to fork this and modify or ask for access to this project as a contributor and make a change?

**Comment by DanielKucal** [comment]
Indeed, that's something that should never happen. I think html should be parsed the same way as is in the options. I've created a pull request: https://github.com/valor-software/ng2-select/pull/490
